docs/guides/getting-started/setup/next-js.mdx

```diff
@@ -0,0 +1,198 @@
+---
+description: Build a Tauri app using Next.js as the frontend build tool
+pagination_next: guides/development/development-cycle
+sidebar_position: 2
+sidebar_custom_props:
+  doc_card_image: /img/guides/getting-started/setup/nextjs-light.svg
+  doc_card_image_dark: /img/guides/getting-started/setup/nextjs-dark.svg
+---
+
+import TauriInit from './_fragments/_tauri-init.mdx'
+import Commands from './_fragments/_commands.mdx'
+import Intro from './_fragments/_intro.mdx'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import Command, { InstallTauriApi } from '@theme/Command'
+
+# Next.js
+
+This guide will walk you through creating your first Tauri app using the React framework [Next.js].
+
+<Intro />
+
+Here's a preview of what we will be building:
+
+![Application Preview](./assets/next-js/next-js-light.png#gh-light-mode-only)
+![Application Preview](./assets/next-js/next-js-dark.png#gh-dark-mode-only)
+
+## Create the Frontend
+
+[Next.js] is a React Framework that comes with both Server-Side Rendering (SSR) and Static-Site Generation (SSG) capabilities. To make Next.js work with Tauri we are going to use the SSG mode since it generates only static files that can be included in the final binary.
+
+Next.js comes with a scaffolding utility similar to [`create-tauri-app`] that can quickly setup a new project from a number of pre-defined templates.
+For this guide we will use the TypeScript template to create a simple project.
+
+<Tabs groupId="package-manager">
+  <TabItem value="npm">
+
+```shell
+npx create-next-app@latest --use-npm --typescript
+```
+
+  </TabItem>
+  <TabItem value="Yarn">
+
+```shell
+yarn create next-app --typescript
+```
+
+  </TabItem>
+  <TabItem value="pnpm">
+
+```shell
+pnpm create next-app --use-pnpm --typescript
+```
+
+  </TabItem>
+</Tabs>
+
+1. _Project name_  
+   This will be the name of your project. It corresponds to the name of the folder this utility will create but has otherwise no effect on your app. You can use any name you want here.
+
+When starting the frontend via the `next-app` command, Next.js will look for a config file named `next.config.js` inside the project root.
```

Comment:
```suggestion
When starting or building the frontend, Next.js will look for a config file named `next.config.js` inside the project root.
```
it's not next-app, but dev/build/export/start, this is the best idea i had to somehow make that clearer.